### PR TITLE
Initial answers

### PR DESCRIPTION
**Pull request by VirtualMe64**

### CODE REVIEW

**Review by ArchWand — COMMENTED** [review]
Good answers

**Review by ArchWand — COMMENTED** [review]
Should have units

File: 4.md [hunk 1]
```diff
@@ -13,25 +13,25 @@ the rows. If there is a miss, please specify the type of miss in the table.
...
-| 0x07           |                    |              |
-| 0x03           |                    |              |
-| 0x02           |                    |              |
-| 0x04           |                    |              |
+| 0x03           |         3           |      Miss (compulsory)        |
+| 0x04           |         4           |      Miss (compulsory)        |
+| 0x07           |         2           |      Miss (compulsory)        |
+| 0x03           |         3           |      Hit        |
+| 0x02           |         2           |      Miss (compulsory)        |
+| 0x04           |         4           |      Miss (compulsory)        |
```
Comment (ArchWand) [inline 4.md:34]: q4.1b minor error

File: 5.md [hunk 2]
```diff
@@ -6,14 +6,16 @@ The CS 2200 TA team is tasked with designing a 4KB (1KB = 1024 bytes) cache.
...
-some drawbacks of his design?
+some drawbacks of his design?  
+This is a direct mapped cache. A drawback is that each address only corresponds to one spot in the cache. As a result, it can be an inefficient use of space, since there’s no flexibility to place addresses in free locations.
 
 ### Q5.2 (3 points)
 Ayush decides to design the cache with 256 ways and 16B cache blocks. What is
 another name for Ayush’s cache design? Why might it not perform as well as
-Nityam’s design?
+Nityam’s design?  
+This is a fully associative cache. The downside is that you need a lot of additional hardware for the comparators, which could greatly slow down lookups.
```
Comment (ArchWand) [inline 5.md:16]: this doesn't quite answer the question, but it's a pretty bad question

File: 6.md [hunk 3]
```diff
@@ -20,24 +20,30 @@ However, Prasad never took CS 2200 and is thus completely lost! Help him by
...
+log_2(512) = 9
 
-c) Tag
+c) Tag  
+64 - 9 - 6 = 49
 
 ### Q6.3 (2 points)
-How many dirty bits are needed per cache block?
+How many dirty bits are needed per cache block?  
+**8**: 1 bit for each 8 bit word
```
Comment (ArchWand) [inline 6.md:40]: byte, not bit

**Comment by ArchWand** [comment]
Overall correct, except for whatever the heck is still going on with q1 (discussion ongoing still). There's a few formatting things I want to change, so I'll pull this to a different branch, then merge into main from there.

Thank you for writing this up.

File: 6.md (hunk 3 above)
Comment (minovermax) [reply 6.md:40]: For #6.5, within the discord I believe it was 20480?

**Pull request merged**